source/constructs/api/data_source/service.py

```diff
@@ -1384,3 +1562,26 @@ def __delete_account(account_id: str, region: str):
         crud.delete_account_by_region(account_id=account_id, region=region)
     except Exception:
         logger.error(traceback.format_exc())
+
+
+def query_glue_connections(account: schemas.AdminAccountInfo):
+    return boto3.client('glue',
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The API method `get_connections` returns paginated results instead of all results. Consider using the pagination API or checking one of the following keys in the response to verify that all results were returned: `IsTruncated`, `NextToken`.

[Learn more](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/paginators.html)

Reply:
Will be resolved in later version.